### PR TITLE
Success message added on successful setup of backend

### PR DESCRIPTION
**Pull request by adnanraghib1308**
Success message added on successful setup of backend. This is the solution for the issue #105

### CODE REVIEW

File: local-test.js [hunk 1]
```diff
@@ -35,3 +36,5 @@ if (!firestoreData) {
   }
   await docRef.delete()
 })()
+
+logger.info('SUCCESS!!! backend setup successful')
```
Comment (swarajpure) [inline local-test.js:40]: Are you sure this is on the right line? Have you tried it out?
Shouldn't it be inside the async function?

Comment (adnanraghib1308) [reply local-test.js]: Yeah right, it should be inside async function. Required changes done.

File: local-test.js [hunk 2]
```diff
@@ -35,6 +35,5 @@ if (!firestoreData) {
     throw new Error('Problem with permission of read and write.\nCheck your firestore permissions')
   }
   await docRef.delete()
+  logger.info('SUCCESS!!! backend setup successful')
```
Comment (swarajpure) [inline local-test.js:38]: Great! Just one suggestion:
Can we have it in a form language? More formal and professional?

`Success! The backend was successfully set up.`

This is just one suggestion, you can have something of your own as well.

Comment (adnanraghib1308) [reply local-test.js]: Sure!
Changes are done. Thanks

**Review by swarajpure — APPROVED** [review]
Great! Thank you for the quick fix! 🎉

**Pull request merged**